app/assets/javascripts/components/itsi_authoring/section_editor_element.js.coffee

```diff
@@ -9,8 +9,8 @@ modulejs.define 'components/itsi_authoring/section_editor_element',
     getInitialState: ->
       selected: not @props.data.is_hidden
 
-    selected: ->
-      selected = (React.findDOMNode @refs.checkbox).checked
+    selected: (e) ->
+      selected = e.target.checked
```

Comment:
This was changed so that the simulated React events in the Jasmine tests would work.